src/libs/getDecodedURIFromDocumentPicker/index.ios.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Returns the decoded URI String. React-native-document-picker returns encoded file path on iOS.
```

Comment:
Let's mention in this comment that this issue exists with react-native-pdf
```suggestion
 * Returns the decoded URI String. react-native-pdf fails to parse the files with spaces on iOS.
```
```

```